crates/models/src/materializations.rs

```diff
@@ -50,6 +50,8 @@ pub enum MaterializationEndpoint {
     Connector(ConnectorConfig),
     /// # A local command (development only).
     Local(LocalConfig),
+    /// # A Dekaf connection
+    Dekaf(DekafConfig),
```

Comment:
Just my opinion here, but "dekaf" doesn't seem like a term that we should make users learn. `kafka` or `kafkaRead` might be better, since "kafka" is the term that people are already familiar with. IDK if that's open for debate, but just my 2 cents.

Reply:
Yeah, I struggled with this a bit, but now that you bring it up I think you're right. I went with Dekaf originally because it made it clear that this was a hard-coded config specifically for our special snow-flake endpoint type. But `kafka` or `kafkaRead` both seem equally descriptive and less confusing 👍 I'll switch the wording around

Reply:
I'm on the other side of this. I think `dekaf` is better because there's far less potential for confusion: if a newcomer was looking at a flow.yaml spec and saw a `kafka` stanza they could easily have a **very** different assumption about what's happening.

"Dekaf" is also something we're explicitly building brand around, and these will be set up in the UI mostly so discover-ability isn't a real concern.

Reply:
☝️ this is the thought process I originally had, will keep it as is then